tests/integration/olp-cpp-sdk-dataservice-read/VersionedLayerClientTest.cpp

```diff
@@ -24,6 +24,7 @@
 #include <matchers/NetworkUrlMatchers.h>
 #include <mocks/NetworkMock.h>
 #include <olp/authentication/Settings.h>
+#include <olp/core/cache/DefaultCache.h>
```

Comment:
Are you sure this include required?

I don't see usage of `DefaultCache` or other members of the _DefaultCache.h_ file.

Reply:
Yes, you're right! This is leftovers.